GE-Ontology/ontology/GE.sadl

```diff
@@ -0,0 +1,250 @@
+    SoftwareUnitTestResult is a type of TEST_RESULT.
+      wasGeneratedBy of SoftwareUnitTestResult only has values of type SoftwareUnitTestExecution.
+
+      //Model Location: DevelopmentPlan.04-SoftwareVerification.StructuralCoverageReport
+    StructuralCoverageReport is a type of ANALYSIS_OUTPUT.
+      wasGeneratedBy of StructuralCoverageReport only has values of type StructualCoverageAnalysis.
+
+        //Model Location: DevelopmentPlan.04-SoftwareVerification.01-SoftwareReviewProcess.SoftwareCodeReview
+      SoftwareCodeReview is a type of REVIEW.
+        turnstile:wasInformedBy of SoftwareCodeReview has values of type SoftwareDesignReview.
+        turnstile:wasInformedBy of SoftwareCodeReview has values of type SoftwareCoding.
+
+       //Model Location: DevelopmentPlan.04-SoftwareVerification.01-SoftwareReviewProcess.SoftwareDesignReview
+      SoftwareDesignReview is a type of REVIEW.
+        turnstile:wasInformedBy of SoftwareDesignReview has values of type SoftwareRequirementsReview.
+        turnstile:wasInformedBy of SoftwareDesignReview has values of type SoftwareDesign.
+
+        //Model Location: DevelopmentPlan.04-SoftwareVerification.01-SoftwareReviewProcess.SoftwareRequirementsReview
+      SoftwareRequirementsReview is a type of REVIEW.
+        turnstile:wasInformedBy of SoftwareRequirementsReview has values of type SoftwareRequirementsDefinition.
+
+        //Model Location: DevelopmentPlan.04-SoftwareVerification.02-SoftwareTestGenerationProcess.DevelopComponentTests
+      DevelopComponentTests is a type of TEST_DEVELOPMENT.
+        turnstile:wasInformedBy of DevelopComponentTests has values of type DevelopUnitTests.
+        turnstile:wasInformedBy of DevelopComponentTests has values of type SoftwareRequirementsReview.
+
+        //Model Location: DevelopmentPlan.04-SoftwareVerification.02-SoftwareTestGenerationProcess.DevelopUnitTests
+      DevelopUnitTests is a type of TEST_DEVELOPMENT.
+        turnstile:wasInformedBy of DevelopUnitTests has values of type SoftwareDesignReview.
+
+        //Model Location: DevelopmentPlan.04-SoftwareVerification.04-SoftwareAnalysis.ControlCouplingAnalysis
+      ControlCouplingAnalysis is a type of ANALYSIS.
+        turnstile:wasInformedBy of ControlCouplingAnalysis has values of type SoftwareComponentTestExecution.
+
+        //Model Location: DevelopmentPlan.04-SoftwareVerification.04-SoftwareAnalysis.DataCouplingAnalysis
+      DataCouplingAnalysis is a type of ANALYSIS.
+        turnstile:wasInformedBy of DataCouplingAnalysis has values of type SoftwareComponentTestExecution.
+
+        //Model Location: DevelopmentPlan.04-SoftwareVerification.04-SoftwareAnalysis.StructualCoverageAnalysis
+      StructualCoverageAnalysis is a type of ANALYSIS.
+        CoveredNodes describes StructualCoverageAnalysis with values of type int.
+        UncoveredNodes describes StructualCoverageAnalysis with values of type int.
```

Comment:
These two properties should be lowercased

Reply:
Thanks for catching these! Will modify.